cargo-near-build/src/pretty_print.rs

```diff
@@ -21,3 +23,13 @@ pub fn step(msg: &str) {
 pub fn success(msg: &str) {
     eprintln!("{} {}", "✓".bold().green(), msg);
 }
+
+pub fn duration(start: Instant, activity: &str) {
+    let duration = std::time::Duration::from_secs(start.elapsed().as_secs());
```

Comment:
`elapsed()` returns you the duration already, what is the point of this `as_secs()` -> `Duration::from_secs()`?

Reply:
this is rounding down to seconds precision, removing smaller fractions (milli, micro and nanoseconds)